benchmarks/run_all.dart

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+import 'dart:math';
+
+import 'latency.dart';
+import 'throughput.dart';
+
+List<int> throughputParserHelper(String aotThroughputStdout) {
+  var aotThroughput = List<int>.filled(2, 0);
+  aotThroughputStdout.split('\n').forEach((line) {
+    final match = RegExp(r'\d+').allMatches(line);
+    if (match.length > 1) {
+      if (int.parse(match.last.group(0)!) > aotThroughput[1]) {
+        aotThroughput[0] = int.parse(match.first.group(0)!);
+        aotThroughput[1] = int.parse(match.last.group(0)!);
+      }
+    }
+  });
+  return aotThroughput;
+}
+
+void main(List<String> args) async {
+  var url = 'https://example.com';
+  var throughputPrallelLimit = 10;
+  switch (args.length) {
+    case 1:
+      url = args[0];
+      break;
+    case 2:
+      url = args[0];
+      throughputPrallelLimit = int.parse(args[1]).toInt();
```

Comment:
It's more common to use flags for arg parsing so that you can provide them in any order and in any combination.

e.g. If I want to just provide a different throughputPrallelLimit I don't want to type http://example.com